### PR TITLE
Twitter text: append creator only if data-title not set.

### PR DESCRIPTION
**Pull request by ksuess**
https://github.com/heiseonline/shariff/issues/327

### CODE REVIEW

**Comment by ksuess** [comment]
Hi @compeak , please be so kind an make a review. 
thank you

File: src/js/shariff.js [hunk 1]
```diff
@@ -152,9 +152,12 @@ class Shariff {
   }
 
   getTitle() {
-    let title = this.getOption('title') || this.getMeta('DC.title')
-    let creator = this.getMeta('DC.creator')
-    if (title && creator) title = `${title} - ${creator}`
+    let title = this.getOption('title')
+    if (!$(this.element).data()['title']) {
```
Comment (compeak) [inline src/js/shariff.js]: Using early return would reduce the indentation level:

```js
if ($(this.element).data()['title']) return title
```

File: src/js/shariff.js [hunk 2]
```diff
@@ -152,9 +152,12 @@ class Shariff {
...
+      title = title || this.getMeta('DC.title')
+      let creator = this.getMeta('DC.creator')
+      if (title && creator) title = `${title} - ${creator}`
```
Comment (compeak) [inline src/js/shariff.js]: There is no need to assign this to `title`:

```js
if (title && creator) return `${title} - ${creator}`
```

**Comment by compeak** [comment]
Thanks.

**Pull request merged**